src/amqproxy/cli.cr

```diff
@@ -1,3 +1,4 @@
+require "./config"
 require "./version"
```

Comment:
`require "./version"` should probably be moved to `config.cr` since that's where `AMQProxy::VERSION` is used?